chrome/content/zotero/lookup.js

```diff
@@ -201,6 +203,23 @@ var Zotero_Lookup = new function () {
 		}
 		return true;
 	}
+
+	this.onFocusOut = function(event) {
+		/*
+			if the lookup popup was triggered by the lookup button,
+			we want to return there on focus out. So we check 
+			(1) that we came from a button and (2) that
+			event.relatedTarget === null, i.e. that the user hasn't used
+			the mouse or keyboard to select something, and focus is leaving
+			the popup because the popup was hidden/dismissed.
+		*/
+		if (this._button && event.relatedTarget === null) {
+			event.preventDefault();
+			event.stopPropagation();
+			this._button.focus();
+			this._button = null;
+		} else this._button = null;
```

Comment:
Nit: Always include braces unless `if` and `else` are both single-line conditionals (which is rare)

```javascript
}
else {
    this._button = null;
}
```